### PR TITLE
Use Capital Framework colors

### PR DESCRIPTION
**Pull request by cfarm**
Removes brand-palette and cf-theme-overrides from the main.less file so that colors come from Capital Framework instead. 

## Additions

-

## Removals

- delete cf-theme-overrides

## Changes

- upgrades to latest Capital Framework components
- add font variables to main.less (these were in cf-theme-overrides, they are not default in Capital Framework yet, but that change is on the way)
- add icon font path to main.less (this was also in cf-theme-overrides)
- remove import of brand-palette.less. for now this is still in project until we can resolve how to manage it since designers update colors here.

## Testing

1. `git fetch && git checkout use-capital-framework-colors`
1. `./setup.sh`
1. `npm start`
1. Confirm brand colors appear correctly at http://localhost:4000/page-components/page-components.html and localhost:4000/brand-guidelines/color-principles.html

## Review

- @user

[Preview this PR without the whitespace changes](?w=0)

## Notes

-

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by cfarm** [comment]
@anselmbradford please ignore the travis build. I just enabled travis for this repo last week to test out some scripts on another branch; we don't need anything to pass for this to be merge-able.

File: src/static/css/main.less [hunk 1]
```diff
@@ -15,11 +15,16 @@
...
 
-// Override the Capital Framework theme variables with colors from the brand color palette.
-@import (less) "cf-theme-overrides.less";
+@webfont-regular: 'AvenirNextLTW01-Regular';
+@webfont-italic: 'AvenirNextLTW01-Italic';
+@webfont-medium: 'AvenirNextLTW01-Medium';
+@webfont-demi: 'AvenirNextLTW01-Demi';
+
+// Icon font path
+@cf-icon-path:                  '../fonts';
```
Comment (anselmbradford) [inline src/static/css/main.less:27]: TRIVIAL: This value doesn't need the huge indent, since it's not aligned with anything.

**Review by anselmbradford — APPROVED** [review]
Works as advertised! 👍

**Review by jimmynotjim — APPROVED** [review]
I can dig it 👍

**Pull request merged**